lib/src/main/java/com/futurewei/alcor/common/rbac/StrictRbacManager.java

```diff
@@ -122,10 +122,14 @@ public void checkCreate(String resourceName, TokenEntity tokenEntity, List<Strin
     private List<String> excludeFields(String actionName, TokenEntity tokenEntity, OwnerChecker ownerChecker) {
         List<String> excludeFields = new ArrayList<>();
         Optional<ActionRbacRule> rbacRuleOptional = serviceRbacRule.getActionRbacRule(actionName);
+        // NOT_JAVA_11_COMPATIBLE: doesn't work in IntelliJ IDE
         if (rbacRuleOptional.isEmpty()) {
```

Comment:
@pkommoju same as here, you need to set Java to 11.

Reply:
same problem.